webapp/src/main/webapp/src/app/components/score-treemap/score-treemap.component.ts

```diff
@@ -177,26 +176,26 @@ export class ScoreTreemapComponent implements OnInit {
     let rgbColor3 = {red: 204, green: 0, blue: 0};
     //background-image: linear-gradient(to left, #3f9c35, #efaa00, #cc0000);
     */
-    let rgbColor1 = {red: 222, green: 243, blue: 255};    
-    let rgbColor2 = {red: 57, green: 165, blue: 220};
-    let rgbColor3 = {red: 0, green: 67, blue: 104};
-    //background-image: linear-gradient(to left, #def3ff, #39a5dc, #004368);
-    var color1 = rgbColor1;
-    var color2 = rgbColor2;
-    var fade = fadeFraction * 2;
+    const rgbColor1 = {red: 222, green: 243, blue: 255};
+    const rgbColor2 = {red: 57, green: 165, blue: 220};
+    const rgbColor3 = {red: 0, green: 67, blue: 104};
```

Comment:
Shouldn't we put these const declarations outside of the function?

Reply:
We can, it's a good idea to store reusable constants outside of component. Do you want me to make changes in this pull request or as a separate one?